components/onfleet/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/onfleet",
-  "version": "0.0.3",
+  "version": "0.0.4",
```

Comment:
I think this should be `0.1.0` [since there are new components being introduced](https://pipedream.com/docs/components/guidelines/#versioning)